lib/t/matches.ex

```diff
@@ -180,6 +202,16 @@ defmodule T.Matches do
     |> Repo.all()
     |> preload_match_profiles(user_id)
     |> with_timeslots(user_id)
+    |> with_expiration_date()
+  end
+
+  @spec list_expired_matches(uuid) :: [%Match{}]
```

Comment:
returning expiredMatches as a separate category in feedChannel.join (likes + matches + missed_calls + expired_matches)